type-definitions/immutable.js.flow

```diff
@@ -390,7 +400,7 @@ declare class SetSeq<T> extends Seq<T,T> mixins SetIterable<T> {
 }
 
 declare class List<T> extends IndexedCollection<T> {
-  static <T>(iterable?: ESIterable<T>): List<T>;
+  static (iterable?: ESIterable<T>): List<T>;
```

Comment:
Gotcha, was just noticing `static of<T>`  right below this one - was just curious why you decided to edit this in this diff, but not any others like it.